qa/full-cluster-restart/build.gradle

```diff
@@ -74,6 +74,7 @@ for (Version bwcVersion : BuildParams.bwcVersions.indexCompatible) {
 
 configurations {
   testArtifacts.extendsFrom testRuntime
+  testArtifacts.extendsFrom testImplementation
```

Comment:
If we just used `testRuntimeClasspath` wouldn't that work as well?

Reply:
That was my initial thought too. but that drags in runtimeOnly which is too much and causes.

Reply:
we probably do not need / want testRuntime at all.